### ISSUE TITLE
Add example or explanation on how to interconnect ns endpoints (nse)

### ISSUE THREAD

**Issue by lewisfelix** [networkservicemesh/deployments-k8s#478]
Good afternoon:

With this new approach for NSM, I have noticed that you have uploaded several cases of connectivity between NSC and NSE. However, there is no example on how to interconnect different NSEs in order to perform a bigger/more complex service with NSM in this case. 

If possible, do you consider adding an example of such characteristics to this repository? In my humble opinion, I think it could be a very positive addition to the examples that are already present.

Thanks!

**Comment by edwarnicke** [comment]
@lewisfelix Good point... which would be the most interesting first example of that in your opinion?

**Comment by lewisfelix** [comment]
@edwarnicke are you referring to those already implemented in the other NSM "examples" repository or in general? :)

In my opinion, one "simple" example (quoted because this is just simple in my mind now, I might be wrong in this one) could be using two "simple-clients" and two "simple-endpoints", interconnecting these later ones to allow both clients to be pinged though the endpoints (i.e. forcing the traffic to traverse the endpoints from one client to another).

If it is the case of those already developed for NSM, perhaps the VPN one seems to the most representative one of all, as it is the one presented in both the main repository and the website, clearly showing how multiple endpoints can cooperate to build a complex service using NSM.

Again Im just theorising here so what I am saying, or considering, now might be wrong or not the best approach for this, but I think an example of such characteristics can be potentially interesting for these new approach to NSM.

Thanks!

**Comment by denis-tingaikin** [comment]
Waiting for moving this from `Backlog` into `To do` by @edwarnicke
